### ISSUE TITLE
Type of ASFF Compliance.Status seems to be wrong

### ISSUE THREAD

**Issue by bebold-jhr** [awsdocs/aws-security-hub-user-guide#14]
I wanted to create an AWS EventBridge rule for AWS Security Hub. It is stated [here](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-event-formats.html) that the filter properties follow the [AWS Security Finding Format (ASFF)](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html)

I wanted to filter for `Compliance.Status`. [This documentation](https://docs.aws.amazon.com/securityhub/latest/userguide/asff-compliance.html) says that `Compliance.Status` is a `String` (with pre-defined values, so basically an `ENUM`). The example features a `String`. However if I try to create a rule in AWS EventBridge with the following event pattern:

```json
{
  "source": [
    "aws.securityhub"
  ],
  "detail-type": [
    "Security Hub Findings - Imported"
  ],
  "detail": {
    "findings": {
      "RecordState": [
        "ACTIVE"
      ],
      "Workflow": {
        "Status": [
          "NEW"
        ]
      },
      "Compliance": {
        "Status": "FAILED"
      }
    }
  }
}
```

I get the following error message:
> Event pattern is not valid. Reason: "Status" must be an object or an array at [Source: (String)"{ "source": [ "aws.securityhub" ], "detail-type": [ "Security Hub Findings - Imported" ], "detail": { "findings": { "RecordState": [ "ACTIVE" ], "Workflow": { "Status": [ "NEW" ] }, "Compliance": { "Status": "FAILED" } } } }"; line: 19, column: 20]

So if I change `Compliance.Status` to an array, the rule is successfully created.

```json
{
  "source": [
    "aws.securityhub"
  ],
  "detail-type": [
    "Security Hub Findings - Imported"
  ],
  "detail": {
    "findings": {
      "RecordState": [
        "ACTIVE"
      ],
      "Workflow": {
        "Status": [
          "NEW"
        ]
      },
      "Compliance": {
        "Status": ["FAILED"]
      }
    }
  }
}
```

So from my perspective the documentation is not accurate here.
Can you confirm this? Is the behavior different in a different context apart from AWS EventBridge?

**Comment by bebold-jhr** [comment]
Hi @prakaprjAWS,

I saw that you first merged #15, but then reverted it in #20. So I assume that the type is actually meant to be a `String`?
If so, regarding my error using the example above, is there a problem in EventBridge?

**Comment by prakaprjAWS** [comment]
It is meant to be a `string` in ASFF, but it seems like when you are creating a custom action in EventBridge, you need to specify it has an `array`. I can understand the confusion there and will consult with our engineering team for options there. Thank you for your question!
